docs/sources/flow/reference/components/prometheus.exporter.kafka.md

```diff
@@ -0,0 +1,103 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.exporter.kafka
+---
+
+# prometheus.exporter.kafka
+The `prometheus.exporter.kafka` component embeds
+[kafka_exporter](https://github.com/davidmparrott/kafka_exporter/v2/exporter) for collecting metrics from a kafka server.
```

Comment:
this link gives a 404

Reply:
Yes, I will modify to the correct address